p2p/transport/webrtc/transport.go

```diff
@@ -78,7 +78,7 @@ const (
 const (
 	DefaultDisconnectedTimeout = 20 * time.Second
 	DefaultFailedTimeout       = 30 * time.Second
-	DefaultKeepaliveTimeout    = 15 * time.Second
+	DefaultKeepaliveTimeout    = 5 * time.Second
 )
```

Comment:
I think it's better to have 4 keep alives before timing out than just 1 as before.